build.sbt

```diff
@@ -87,7 +87,7 @@ lazy val lorre = (project in file("conseil-lorre"))
     javaExtras = Seq("-Xmx512M", "-Xss1M", "-XX:+CMSClassUnloadingEnabled")
   )
   .enableAssembly()
-  .dependsOn(common, commonTestKit % Test)
+  .dependsOn(common % "compile->compile;test->test", commonTestKit % Test)
```

Comment:
Can you shortly explain what didn't work? Thanks

Reply:
without it I can't use `DbFixtures` from common in Lorre tests